templates/viewer.php

```diff
@@ -289,10 +289,13 @@
                 <div class="verticalToolbarSeparator hiddenMediumView"></div>
 
                 <div id="editorModeButtons" class="splitToolbarButton toggled" role="radiogroup" hidden="true">
-                  <button id="editorFreeText" class="toolbarButton" disabled="disabled" title="Text" role="radio" aria-checked="false" tabindex="34" data-l10n-id="editor_free_text2">
+                  <button id="editorStamp" class="toolbarButton hidden" disabled="disabled" title="Image" role="radio" aria-checked="false" tabindex="34" data-l10n-id="editor_stamp">
+                    <span data-l10n-id="editor_stamp_label">Image</span>
+                  </button>
+                  <button id="editorFreeText" class="toolbarButton" disabled="disabled" title="Text" role="radio" aria-checked="false" tabindex="35" data-l10n-id="editor_free_text2">
```

Comment:
Isnt this missing here?
```suggestion
                  <button id="editorFreeText" class="toolbarButton" disabled="disabled" title="Text" role="radio" aria-checked="false" aria-controls="editorFreeTextParamsToolbar" tabindex="35" data-l10n-id="editor_free_text2">
```

Reply:
Good catch. That actually comes from [a different version bump](https://github.com/nextcloud/files_pdfviewer/commit/3419664b5f8222c2ee272d588af349f3dbc0c277#diff-b0e73f4391c0f98303bb616d5a1b8d7a085a5924a37edd31a84abbb9940e7cf5); I have adjusted that in its own commit.

Reply:
Maybe we should create a PR that gets red or something whenever the viewer.html gets updated so that we can adjust our template as well?

Reply:
Sounds good :-) But I am not knowledgeable on GitHub workflows... :shrug: Could you do that? :-) Thanks!

Otherwise feel free to open an issue about that.

Reply:
yes, I can do that :)

Reply:
https://github.com/nextcloud/files_pdfviewer/pull/810